litellm/utils.py

```diff
@@ -3543,6 +3543,12 @@ def _select_tokenizer(model: str):
     elif "llama-2" in model.lower():
         tokenizer = Tokenizer.from_pretrained("hf-internal-testing/llama-tokenizer")
         return {"type": "huggingface_tokenizer", "tokenizer": tokenizer}
+    elif model in litellm.mistral_tokenizer_models:
+        from mistral_common.tokens.tokenizers.mistral import MistralTokenizer
```

Comment:
Hey @Dev-Khant from where is this package being installed? 

it would be helpful - if a user is missing this package - to tell them how to install it - e.g. https://github.com/BerriAI/litellm/blob/e52e4cc1a97e8b9f0394512eae52404098787ee2/litellm/integrations/langfuse.py#L20

Reply:
we already install huggingface_hub (litellm dep) 

if the tokenizer is on huggingface, it would be easier to just use that since we can guarantee the package exists in their environment

Reply:
Found it - https://huggingface.co/Xenova/mistral-tokenizer-v3

Could we use this instead?

Reply:
Alright, I'll handle add a case to install the package if not found. But this https://huggingface.co/Xenova/mistral-tokenizer-v3 only has 1 tokenizer while the library `mistral-common` supports 6 different tokenizers.

So do you want me to add the HF one only? @krrishdholakia